### PR TITLE
Update geometric.js with Math.round

### PR DESCRIPTION
**Pull request by zhoutuo**
use `Math.round` instead of `Math.floor` for `floor` makes small increments rounded back to zero.

### CODE REVIEW

File: src/algorithms/geometric.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ export default {
   },
 
   getValue(x, min, max) {
-    return (Math.floor(((x / 100) ** 2) * (max - min)) + min);
```
Comment (ljharb) [inline src/algorithms/geometric.js]: I'm not sure I understand the use case; could you provide tests that would fail without this change?

**Comment by ljharb** [comment]
Looks like this is duplicated by #131.

**Comment by zhoutuo** [comment]
@ljharb yep, i failed to pull this onto my laptop and closed this.

**Comment by ljharb** [comment]
@zhoutuo np :-) if that happens in the future, please ping me on slack and i can walk you through how to fix it

**Comment by zhoutuo** [comment]
@ljharb took a while that i dont have access to origin and all that. :)

**Pull request closed without merging**